pybullet_toolkit/robots/ur5_rg2.py

```diff
@@ -16,9 +16,9 @@ class UR5_RG2(object):
   '''
   def __init__(self):
     # Setup arm and gripper variables
-    self.max_forces = [150, 150, 150, 28, 28, 28, 100, 100]
-    self.gripper_close_force = [100] * 2
-    self.gripper_open_force = [100] * 2
+    self.max_forces = [150, 150, 150, 28, 28, 28, 500, 500]
+    self.gripper_close_force = [500] * 2
+    self.gripper_open_force = [500] * 2
```

Comment:
There a reason you set the force to 500?

Reply:
not really?? just some random trials I guess... I can switch it back to 100